doc/src/pair_3b_table.rst

```diff
@@ -0,0 +1,265 @@
+.. index:: pair_style 3b/table
+
+pair_style 3b/table command
+===========================
+
+Syntax
+""""""
+
+.. code-block:: LAMMPS
+
+   pair_style style
+
+* style = *3b/table*
+
+
+Examples
+""""""""
+
+.. code-block:: LAMMPS
+
+   pair_style 3b/table
+   pair_coeff * * spce.3b type
+   pair_coeff * * GaN.3b Ga N Ga
+
+
+Description
+"""""""""""
+
+The *3b/table* style is a pair style for generic tabulated three-body
+interactions.  It has been developed for (coarse-grained) simulations
+(of water) with Kernel-based machine learning (ML) potentials
+(:ref:`Scherer2 <Scherer2>`).  As for the pair style :doc:`pair_style sw
+<pair_sw>` or :doc:`pair_style sw/3b/table <pair_sw_3b_table>`, the energy of
+a system is computed as a sum over three-body terms:
```

Comment:
What I saw is that I forgot to to transfer the read in energies from the table column. I added the corresponding line now into the function Pair3BTable::uf_lookup( ). Now, the issue with the uninitialized data should be solved. It is true that in the examples the energy column of the tables is zero. That was due to the parametrization, but the pair style should allow for correct energy treatment.

Reply:
> It is true that in the examples the energy column of the tables is zero. That was due to the parametrization, but the pair style should allow for correct energy treatment.
Would it be possible to change the example to one that *does* include the energy? Otherwise we should add a README file explaining it to the examples folder. Otherwise people might just use the table files for their own purposes and expect that there are meaningful energies and get inconsistent results (or worse).

Reply:
Sure, in principle yes. The issue is, honestly, that I/we generated the table with the VOTCA kernel based ML code, where we do generalized force matching and therefore at the moment only can calculate / output forces and not the energies (yet). We tested everything and the MD simulation with the forces works fine and can reproduce results of model systems (as the example) also without the energies. The idea was of course to include the energy column into the table and the pair style though it has not been used so far. Of course, one should comment on this explicitly in the README file! I can do this